### PR TITLE
Add support for Windows default conf paths

### PR DESCRIPTION
**Pull request by jterry75**
Signed-off-by: Justin Terry <jlterry@amazon.com>

### CODE REVIEW

**Comment by jterry75** [comment]
@vineeth-bandi

**Review by mikebrow — APPROVED** [review]
LGTM

File: types_windows.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package cni
+
+const (
+	DefaultNetDir = "C:\\Program Files\\containerd\\cni\\conf"
+	DefaultCNIDir = "C:\\Program Files\\containerd\\cni"
```
Comment (kevpar) [inline types_windows.go:23]: [CRI](https://github.com/containerd/containerd/blob/main/pkg/cri/config/config_windows.go#L32) uses `C:\Program Files\containerd\cni\bin` for this path. Should we do the same here to be consistent?

Comment (kevpar) [reply types_windows.go:23]: IMO let's change to `bin` and then check in :)

Comment (jterry75) [reply types_windows.go]: Done deal!

**Review by kevpar — APPROVED** [review]
LGTM!

**Comment by jterry75** [comment]
Thanks Mike!

**Pull request merged**